tests_e2e/scenarios/runbooks/daily.yml

```diff
@@ -64,19 +48,27 @@ platform:
         marketplace: "$(marketplace_image)"
         vhd: $(vhd)
         location: $(location)
-        name: $(vm_name)
         vm_size: $(vm_size)
 
-testcase:
-  - criteria:
-      area: bvt
+combinator:
+  type: grid
+  items:
+    - name: marketplace_image
+      value:
+        - "Canonical UbuntuServer 18.04-LTS latest"
+        - "Debian debian-10 10 latest"
+        - "OpenLogic CentOS 7_9 latest"
+        - "SUSE sles-15-sp2-basic gen2 latest"
+        - "RedHat RHEL 7-RAW latest"
+        - "microsoftcblmariner cbl-mariner cbl-mariner-1 latest"
+        - "microsoftcblmariner cbl-mariner cbl-mariner-2 latest"
+
+concurrency: 10
+
+notifier:
+  - type: env_stats
+  - type: junit
+
+include:
+  - path: ./include/ssh_proxy.yml
 
-#
```

Comment:
The ssh proxy settings will be shared by multiple runbooks, so I refactored it into its own file